src/strings/ops.c

```diff
@@ -1,6 +1,7 @@
 #include "platform/memmem.h"
 #include "moar.h"
 #define MVM_DEBUG_STRANDS 0
+#define MAX_GRAPHEMES     0xFFFFFFFF
```

Comment:
Can you add a comment here that 0xFFFFFFFF is the largest value that can be stored in MVMStringIndex (32bit-uint)